app/src/main/java/org/mozilla/fenix/browser/BaseBrowserFragment.kt

```diff
@@ -929,7 +932,13 @@ abstract class BaseBrowserFragment : Fragment(), UserInteractionHandler,
             }
         } else {
             view?.let { view ->
-                browserInitialized = initializeUI(view) != null
+                val tab = getCurrentTab()
```

Comment:
Since we have this block twice what do you think about adding a private function here in `BaseBrowserFragment`:

```kotlin
 private fun initializeUI(view: View): Boolean {
  val tab = getCurrentTab()
  return if (tab != null) {
    initializeUI(view, tab)
      true
    } else {
      false
    }
  }
```

Then we can call it here and in `onViewCreated` as:

```kotlin
browserInitialized = initializeUI(view)
```